runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2335,9 +2335,14 @@ MM_CopyForwardScheme::scanReferenceObjectSlots(MM_EnvironmentVLHGC *env, MM_Allo
 	
 	GC_SlotObject referentPtr(_javaVM->omrVM, &J9GC_J9VMJAVALANGREFERENCE_REFERENT(env, objectPtr));
 
-	/* Iteratoring and copyforwarding  the slot reference */
-	if (success) {
-		success = iterateAndCopyforwardSlotReference(env, reservingContext, objectPtr);
+	/* Iteratoring and copyforwarding the slot reference without leaf bit, sacrifice a minor performance to avoid code complication, could optimazation later. */
```

Comment:
Iterating and copyforwarding regular reference slots, except the special (soft) referent slot. Not making use of leaf bit optimization, sacrificing minor performance to avoid code complication. Could optimize later, if/when using ObjectScanner */